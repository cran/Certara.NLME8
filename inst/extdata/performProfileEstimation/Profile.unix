Scenario,Theta,Estimate,LogLik,RetCode,Delta,Percent
performProfileEstimation,tvV,7.954820,1484.464290,1,-2,
performProfileEstimation,tvV,8.954820,1496.766260,1,-1,
performProfileEstimation,tvV,9.954820,1500.947370,1,0,
performProfileEstimation,tvV,10.954820,1497.536670,1,1,
performProfileEstimation,tvV,11.954820,1489.437100,1,2,
