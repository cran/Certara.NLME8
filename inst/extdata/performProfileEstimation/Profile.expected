"Scenario","Theta","Estimate","LogLik","RetCode","Delta","Percent"
"performProfileEstimation","tvV",7.95482,1484.46429,1,-2,NA
"performProfileEstimation","tvV",8.95482,1496.76626,1,-1,NA
"performProfileEstimation","tvV",9.95482,1500.94737,1,0,NA
"performProfileEstimation","tvV",10.95482,1497.53667,1,1,NA
"performProfileEstimation","tvV",11.95482,1489.4371,1,2,NA
